packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts

```diff
@@ -126,14 +126,21 @@ export namespace ZoweExplorerZosmf {
             return zosfiles.Utilities.isFileTagBinOrAscii(this.getSession(), ussFilePath);
         }
 
-        public getContents(inputFilePath: string, options: zosfiles.IDownloadOptions): Promise<zosfiles.IZosFilesResponse> {
+        public getContents(inputFilePath: string, options: zosfiles.IDownloadSingleOptions): Promise<zosfiles.IZosFilesResponse> {
             return zosfiles.Download.ussFile(this.getSession(), inputFilePath, options);
         }
 
         public copy(outputPath: string, options?: Omit<object, "request">): Promise<Buffer> {
             return zosfiles.Utilities.putUSSPayload(this.getSession(), outputPath, { ...(options ?? {}), request: "copy" });
         }
 
+        public async move(oldPath: string, newPath: string): Promise<void> {
```

Comment:
I know we added the breaking change interface-updates for the `MainframeInteraction.IUss`. But I couldn't find an entry in this PR's ZE-API's Changelog about adding the required `move` to `MainframeInteraction.IUss` or changing the signature of `getContents` in `MainframeInteraction.IMvs`.

Again, I could be missing these 😋

Reply:
Added an entry for move, but as for `getContents` I wasn't sure whether that was breaking or not, I'll revisit this once I get to Timothy's feedback 😅

Reply:
I looked into this more and it won't be a breaking change. I added this to the changelog 👍